### PR TITLE
Canvassing Efforts and Scripts

### PR DESCRIPTION
**Pull request by dmitry-ulyanov**

### CODE REVIEW

**Comment by j-ro** [comment]
nice, this is looking really good. A few things:

1. I thought we discussed removing the people and canvassers endpoints from canvass efforts for now? If so, I'd just take that stuff out of the file there, including the examples.

2. For script questions, you'd want a link to the question (as well as embedded, which is optional).

3. Also on script questions, that isn't exactly how OSDI delete works. Rather, you'd delete script questions one at a time by calling delete on their individual endpoints. We don't have a delete collection function, at least not yet anyway.

4. For scripts, you'd want a link to script questions (which is how it is in the example, but not up above in the tables)

We'll also want to think about helpers here, creating scripts with questions perhaps. Not sure.

File: canvassing_efforts.md [hunk 1]
```diff
@@ -0,0 +1,1115 @@
...
+
+
+## Scenarios
+
+{% include scenarios_intro.md %}
+
+### Scenario: Retrieving a collection of Canvassing Effort resources (GET)
+
+Canvassing Effort resources are sometimes presented as collections of canvassing efforts. For example, calling the canvassing_efforts endpoint will return a collection of all the canvassing efforts stored in the system's database associated with your api key.
+
```
Comment (joshco) [inline canvassing_efforts.md]: need to add link to script questions

File: canvassing_efforts.md [hunk 2]
```diff
@@ -0,0 +1,1115 @@
...
+        },
+        "osdi:people" : {
+                "href": "https://osdi-sample-system.org/api/v1/canvassing_efforts/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0bc3/people"
+        },
+        "osdi:script" : {
+                "href": "https://osdi-sample-system.org/api/v1/script/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0ba3"
+        }, 
+         "osdi:canvassers" : {
+            "href":"https://osdi-sample-system.org/api/v1/canvassing_efforts/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0bc3/canvassers"
+         }
```
Comment (joshco) [inline canvassing_efforts.md]: need script_questions link

**Comment by joshco** [comment]
Approved 3/24 with amendment for script questions: 
> Note:  When putting or creating Script Question resources, it is up to the server to validate or return errors when problems with sequence arises

**Pull request merged**